ultra/ultra/scenarios/interface.py

```diff
@@ -53,14 +53,20 @@
         "--task", help="type a task id [0, 1, 2, 3]", type=str
     )
     parser_generate_scenarios.add_argument(
-        "--save-dir", help="directory for saving maps", type=str, default=None
+        "--save-dir", help="directory for saving scenarios", type=str, default=None
     )
     parser_generate_scenarios.add_argument(
         "--root-dir",
-        help="directory for saving maps",
+        help="directory for saving scenarios",
```

Comment:
Good catch, but isn't this the directory where we can find the `task<name>/` folder? Maybe something like:
```suggestion
        help="path to task directory",
```
I think this is what the flag does, but of course, correct me if I'm wrong.